### PR TITLE
Remove kpod code after repository move

### PR DESCRIPTION
**Pull request by mheon**
We've moved kpod development into a separate repository to help speed up kpod development without impacting the stability and merge requirements of CRI-O. This removes the kpod code, which now lives at github.com/projectatomic/libpod

### CODE REVIEW

**Comment by mheon** [comment]
Please hold off on merging this for now, we're still getting tests set up at the new repo. This is just to let the tests run here so we can merge once the new repo is set up properly.

File: README.md [hunk 1]
```diff
@@ -40,38 +40,6 @@ It is currently in active development in the Kubernetes community through the [d
 | Command                                              | Description                                                               | Demo|
 | ---------------------------------------------------- | --------------------------------------------------------------------------|-----|
 | [crio(8)](/docs/crio.8.md)                           | OCI Kubernetes Container Runtime daemon                                   ||
```
Comment (TomSweeneyRedHat) [inline README.md]: I think we should leave a bread crumb somewerein here, even if it's just temporary (few months), saying that kpod has been moved to it's own project at "blah".

**Comment by rhatdan** [comment]
You need to add something into the CRI-O README.md about the relocation of the kpod code.  Also might need to update transfer.md

**Comment by mheon** [comment]
I've added a pointer to the new repository in the readme. I've left transfer.md untouched for now, but I think an argument could be made for removing it here and preserving it in the new libpod repository. @rhatdan @TomSweeneyRedHat WDYT?

**Comment by mheon** [comment]
@mrunalp @runcom We have CI up in the new repo. Once the last few questions are addressed on this PR it should be good to merge.

**Comment by TomSweeneyRedHat** [comment]
LGTM.  I think transfer.md should eventually move to the libpod site too.  Although it would probably be good to have a smaller variant remain here that basically points/links to the one on the kpod site.   I'd rather not have two copies to maintain.  Thanks for the pointer touchup @mheon

**Comment by rhatdan** [comment]
I am going to merge this, since we still have people opening issues on kpod.  Better to move them over to libpod.

**Pull request merged**